public/download.php

```diff
@@ -9,64 +14,71 @@
 <div id="mainpage">
     <div id='fullcontainer'>
 
-        <h2>Downloads</h2>
-
-        <p class="embedded mb-5">
-            <b>Upgrade:</b> If you were brought to this page because the emulator told you
-            that a new version is available, download the new version and
-            extract it over the existing folder. This way you won't lose
-            any save files that you may have.
-        </p>
+        <h2>Emulators supporting RetroAchievements</h2>
 
         <?php foreach ($emulators as $emulator): ?>
             <h2 class="longheader" id="<?= mb_strtolower($emulator['handle'] ?? null) ?>">
-                <a href="#<?= mb_strtolower($emulator['handle'] ?? null) ?>"><?= $emulator['handle'] ?? null ?></a> <small>(<?= $emulator['name'] ?? null ?>)</small>
-            </h2>
-            <div class="mb-1">
-                <?php if ($emulator['description'] ?? false): ?>
-                    <?= nl2br($emulator['description']) ?><br>
+                <a href="#<?= mb_strtolower($emulator['handle'] ?? null) ?>"><?= $emulator['handle'] ?? null ?></a>
+                <?php if ($emulator['handle'] != $emulator['name']): ?>
+                    <small>(<?= $emulator['name'] ?? null ?>)</small>
                 <?php endif ?>
-                <?php if ($emulator['link'] ?? false): ?>
-                    <a class="" href="<?= $emulator['link'] ?>">Documentation</a>
-                <?php endif ?>
-            </div>
-            <div class="mb-3 flex flex-col lg:flex-row justify-between items-start">
-                <div class="flex-1 mb-3">
-                    <?php if (!empty($emulator['systems'])): ?>
-                        <?php sort($emulator['systems']) ?>
-                        <b>Supported Systems:</b><br>
-                        <ul style="column-count: 3">
-                        <?php foreach ($emulator['systems'] as $system): ?>
-                            <?php
-                            sanitize_outputs($system);
-                            ?>
-                            <li>- <?= $system ?></li>
-                        <?php endforeach ?>
-                        </ul>
+            </h2>
+            <div class="flex flex-col lg:flex-row justify-between items-start">
```

Comment:
```suggestion
            <div class="flex flex-col lg:flex-row justify-between items-start mb-6">
```

When the layout is MD or less and in `flex-col` mode, the spacing between each emulator item seems to be fairly tight. This change adds a bit of gap between the elements to improve readability.

**BEFORE**
![Screenshot 2023-04-15 at 6 04 47 PM](https://user-images.githubusercontent.com/3984985/232255103-5a65b757-5fa4-4e3b-b5ec-ae5c777d6d4b.png)

**AFTER**
![Screenshot 2023-04-15 at 6 05 55 PM](https://user-images.githubusercontent.com/3984985/232255132-28f44506-f699-420b-a53b-1387c701ea5a.png)